File: C/Exercicio34.c
#include <stdio.h>
 
int main(int argc, char *argv[]){
  
  int numeros[10];
  
  printf("Digite os 10 numeros: ");
  for(int i=0; i<10; i++){
    scanf("%i", &numeros[i]);
  }
  
  for (int i=1; i<10; ++i) {
    if (numeros[0]<numeros[i]){
      numeros[0] = numeros[i];
    }
  }
  
  printf("Dentre os numeros digitados, o maior foi: %i", numeros[0]);
  
  return 0;
}

File: Python/Exercicio34.py
numeros = []

for i in range(0, 10, 1):
    x = int(input('Digite um numero: '))
    numeros.append(x)

for i in range(0, 10, 1):
    if numeros[0]<numeros[i]:
        numeros[0]=numeros[i]

print("O maior número digitado foi:", numeros[0])